devicon.json

```diff
@@ -2934,6 +2934,28 @@
             }
         ]
     },
+    {
+        "name": "githubcodespaces",
+        "altnames": [],
```

Comment:
```suggestion
        "altnames": ["github codespaces"],
```

Reply:
Can it be "GitHub Codespaces" instead? Or is it case-insensitive?

Reply:
Yeah, agreed, I changed the suggestion so you can apply it by clicking `commit suggestion`

Reply:
Thank you! ✌🏼